packages/by-name/kata/kata-image/package.nix

```diff
@@ -155,7 +155,7 @@ let
       runHook preBuild
 
       # Check if filesystem is ext.*
-      fstype=$(stat -f -c %T .)
+      fstype=$(df -T . | awk '{print $2}' | tail -n 1)
```

Comment:
As far as I understand, the issue with stat is that the superblock magic of ext4 is the same as for ext2 and ext3, so it lacks details to understand the diff and nobody bothered to upgrade the type string (or maybe it needs to stay like that for backwards compatibility). For our purposes, this does not really matter, since all ext* are affected. Could we just replace the `$fstype` in the error message with a literal `ext2/ext3/ext4`?